From 238e0f78c950a940544049cafec63f26c8722250 Mon Sep 17 00:00:00 2023
From: Yan <yanxk.fnst@fujitsu.com>
Date: Fri, 25 Aug 2023 09:59:38 +0800
Subject: [PATCH] boa: fix icky kernel bug

Upstream-Status: Pending
comment out these DIE sentecnce to avoid crash
when you start the boa server.

Signed-off-by: Yan <yanxk.fnst@fujitsu.com>
---
 src/boa.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/boa.c b/src/boa.c
index 8ce2c7b..b2f3f83 100644
--- a/src/boa.c
+++ b/src/boa.c
@@ -222,9 +222,11 @@ static void drop_privs(void)
         /* test for failed-but-return-was-successful setuid
          * http://www.securityportal.com/list-archive/bugtraq/2000/Jun/0101.html
          */
+        #if 0
         if (setuid(0) != -1) {
             DIE("icky Linux kernel bug!");
         }
+        #endif
     } else {
         if (server_gid || server_uid) {
             log_error_time();
--
2.25.1
